ext/drx/drx.c

```diff
@@ -2203,7 +2203,7 @@ drx_expand_scatter_gather(void *drcontext, instrlist_t *bb, OUT bool *expanded)
      * mask register as well as a temp xmm register.
      */
     if (!expand_scatter_gather_drreg_initialized) {
-        drreg_options_t ops = { sizeof(ops), 3, false, NULL, true };
+        drreg_options_t ops = { sizeof(ops), 3 + 3, false, NULL, true };
```

Comment:
Why "3 + 3"?  Either explain in a comment or used named constants whose names make it clear.

Reply:
Done.